src/Symfony/Component/Validator/Constraints/File.php

```diff
@@ -16,6 +16,7 @@
 
 /**
  * @Annotation
+ *
```

Comment:
I know fabbot wants you to add these lines, but that's a false positive for us :)